docs/reference/aggregations/bucket/geodistance-aggregation.asciidoc

```diff
@@ -116,30 +116,7 @@ POST /museums/_search?size=0
 
 <1> The distances will be computed in kilometers
 
-There are two distance calculation modes: `arc` (the default), and `plane`. The `arc` calculation is the most accurate. The `plane` is the fastest but least accurate. Consider using `plane` when your search context is "narrow", and spans smaller geographical areas (~5km). `plane` will return higher error margins for searches across very large areas (e.g. cross continent search). The distance calculation type can be set using the `distance_type` parameter:
-
-[source,console]
---------------------------------------------------
-POST /museums/_search?size=0
-{
-  "aggs": {
-    "rings": {
-      "geo_distance": {
-        "field": "location",
-        "origin": "52.3760, 4.894",
-        "unit": "km",
-        "distance_type": "plane",
-        "ranges": [
-          { "to": 100 },
-          { "from": 100, "to": 300 },
-          { "from": 300 }
-        ]
-      }
-    }
-  }
-}
---------------------------------------------------
-// TEST[continued]
+Field `distance_type` is deprecated and the distance calculation mode is handled internally.
```

Comment:
I don't think we should remove the documentation as we still accept both values and we will calculate differently. Not sure exactly how we handle this, @jrodewig do you have a recommendation here?

Reply:
@iverase when discussed with Nick (a long time ago) I had understood everything is calculated as `Arc` (because of something that had changed in Lucene I think) sorry I'm not clearer on that justification. Moreover, the underlying code seemed to confirm that everything is now `Arc` so I went along with this & just tried to make it consistent. 

*If that is _NOT_ the intention then let me know and I suggest to stop the review now as that will require multiple changes to the PR which I am fine to do*

Reply:
In most cases, we would add a deprecation admonition and leave the existing documentation as is.
For example, I'd add something like this:

```
deprecated:[7.13.0,The `distance_type` parameter will be removed in a future release.] There are two distance...
```

If the parameter now calculates everything as `arc` now, I'd add the deprecation note and update the docs to indicate this.